.gitignore

```diff
@@ -66,9 +66,13 @@ vendor/
 static/vendor/
 src/vendor/
 util.js
-dist/
+/dist/
```

Comment:
```suggestion
/dist/  # docs/dist/ still checked in so GitHub Pages can use it to serve built static assets
```

Reply:
Shortened this to `# docs/dist/ checked in so GitHub Pages can serve built static assets.` to fit within 80 characters.

Reply:
https://github.com/cfpb/capital-framework/pull/908/commits/5af96b34c507771abd738f7ae6bc40470bddf66c